src/styles/index.scss

```diff
@@ -1,3 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  flex-direction: column;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
   margin: 0;
 }
+
+@import 'main';
```

Comment:
The `main` stylesheet is imported here. Verify that this imported file contains styles that match the calendar's starting day as indicated in the HTML (`calendar--start-day-sun`). If not, consider adding the necessary styles here or in the `main` file.